### PR TITLE
Dasd 4748 create yaml build definition

### PR DESCRIPTION
**Pull request by jack-education**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+name: $(TeamProject)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
```
Comment (EwanNoble) [inline azure-pipelines.yml]: This will be overwritten by the GitVersion task so may as well be removed

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+trigger:
```
Comment (EwanNoble) [inline azure-pipelines.yml]: I think it's better to build all branches
```suggestion
trigger:
  batch: true
  branches:
    include:
      - "*"

pr: none
```

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    projects: 'src/**/*.csproj'
+    noCache: true
+
+- task: DotNetCoreCLI@2
+  displayName: dotnet build
+  inputs:
+    projects: 'src/**/*.csproj'
+    arguments: '--configuration $(buildConfiguration)  --no-restore'
+
+- task: alanwales.resharper-code-analysis.custom-build-task.ResharperCli@1
```
Comment (EwanNoble) [inline azure-pipelines.yml]: Was this asked for? We don't use this on the rest of DAS so better to leave it out

Comment (jack-education) [reply azure-pipelines.yml]: Removed alanwales.resharper-code-analysis task.

File: azure-pipelines.yml (hunk 2 above)
Comment (jack-education) [reply azure-pipelines.yml]: Suggestion committed

Comment (jack-education) [reply azure-pipelines.yml]: Unneeded 'master' line removed

**Pull request merged**